### ISSUE TITLE
Develop an SAP Fiori App Using SAP Business Application Studio

### ISSUE THREAD

**Issue by Martin-Gaub** [SAPDocuments/Tutorials#5242]
Tutorial URL: https://developers.sap.com/tutorials/appstudio-fioriapps-create.html

Step 6.

Hi Raz,
Thanks a lot for this tutorial!
Unfortunately, the creation of a Run Configuration is not working for me. I do get the dialog to enter substeps A to D, but after pressing Enter nothing happens. No error message is shown either. 
Any idea what might be the reason? Thanks!
Martin

**Comment by raz-korn** [comment]
Hi Martin.

Can you share which landscape are you using? trial? factory?
I just tried it on my trial and factory accounts and it works well.
Can you share a screenshot of the Run Configurations pane after you go through the creation of the run configuration?

Regards - Raz

**Comment by Martin-Gaub** [comment]
Hi Raz,
Thank you for you reply. I only have a trial account. Something must be wrong or missing in my settings, as absolutely nothing happens after pressing Enter at the end of the dialog. The Run Configurations pane remains totally empty. 
![RunConfigurations](https://user-images.githubusercontent.com/66268535/84595503-892cef80-ae58-11ea-8b27-f04eebf6d65a.PNG)

**Comment by raz-korn** [comment]
Hi @Martin-Gaub ,
I apologize for the late response, needed to do some research on this.
Does it still happen?
If it does:
We're familiar with the root-cause for this and we're investigating the best approach to overcome it. As a workaround, you can right-click the folder fioridemo-approuter and select "Open in Terminal". In the terminal, execute the command "npm install".

Let me know if this solved the issue.

Regards - Raz

**Comment by Martin-Gaub** [comment]
Hi Raz, @raz-korn,
Last weekend my application wouldn't start, that's why I was only now able to try again. 
Surprisingly, I found lots of Run Configurations in my pane! So they did get created but never showed up. I deleted them, created a new one and ran it - everything is working fine now. No need for the workaround. Great!
Suppose you guys have fixed the problem. Thank you very much!
Regards,
Martin
